README.md

```diff
@@ -34,204 +32,39 @@ The command line interface (CLI) allows users to submit their project package de
    Signature and comment signature verified
    Trusted comment: Phylum - the future of software supply chain security
    ```
+   
+3. Run the installer script for [installation](https://docs.phylum.io/docs/installation)
```

Comment:
This is a non-existent link.

Reply:
Removed link until we settle on a release assets strategy